scripts/filter-known-issues.py

```diff
@@ -59,13 +59,13 @@ def config_import_file(filename):
             # finditer() finds all the blocks and selects
             # the bits of mmapped-file that comprises
             # each--we compile it into a regex and append.
-            for m in re.finditer(config_regex, mm, re.MULTILINE):
+            for m in re.finditer(b"config_regex", mm, re.MULTILINE):
```

Comment:
This definitely can't be right. Original code used value of `config_regex` var, changed uses literal bytes string `"config_regex"`. If you want to turn `config_regex`'s value to be bytes, you need to put `b` before its value in initialization (before each "subvalue").